src/main/java/org/bytedeco/gradle/javacpp/BuildPlugin.java

```diff
@@ -131,7 +189,16 @@ boolean isLibraryPath(String path) {
                     JavaCompile.class, new Action<JavaCompile>() { public void execute(JavaCompile task) {
                 task.setSource(main.getJava());
                 task.setClasspath(main.getCompileClasspath());
-                task.setDestinationDir(main.getJava().getOutputDir());
+                setProperty(
+                    "setDestinationDir",
```

Comment:
Could you add some comments for the records for these wrapper calls ? for instance:
```
               setProperty(
                    "setDestinationDir", // Deprecated in 7.1, removed in Gradle 9.0
                    "getDestinationDirectory", // Since 6.1
                    task,
                    getProperty(
                        "getOutputDir", // Deprecated in 7.1, Removed in 8.0
                        "getClassesDirectory", // Since 6.1
                        main.getJava()
                    )
                );
```